src/FileLoader/ImageLoaders/BMPLoader.cs

```diff
@@ -1,13 +1,60 @@
 ﻿using FileLoader.Interfaces;
+using System.IO;
 using System;
+using Shared.Structs;
+using FileLoader.structs;
 
 namespace FileLoader.ImageLoaders
 {
     public class BMPLoader : IImageLoader
     {
         public IImage Load(string path)
+        {            
+            try
+            {
+                var bmpBytes = File.ReadAllBytes(path);
+                var bmpFile = new BMPFile();
+
+                var resolutions = GetBMPResolutions(bmpBytes);
+                bmpFile.Width = resolutions.Item1;
+                bmpFile.Height = resolutions.Item2;
+                bmpFile.Size = GetFileSize(bmpBytes);
+            }
+            catch
+            {
+                throw new NotImplementedException();
```

Comment:
Please create your own `Exception`, place it in `FileLoader/Exceptions` and throw it there instead of `NotImplementedException`. For example `BMPFileLoadingException`.

Dont forget to comment it https://docs.microsoft.com/pl-pl/dotnet/csharp/programming-guide/xmldoc/exception